landlab/components/chi_index/tests/test_chi_finder.py

```diff
@@ -13,3 +13,25 @@ def test_route_to_multiple_error_raised():
 
     with pytest.raises(NotImplementedError):
         ChiFinder(mg, min_drainage_area=1., reference_concavity=1.)
+
+
+def test_bad_reference_area():
+    mg = RasterModelGrid((10, 10))
+    z = mg.add_zeros("node", "topographic__elevation")
+    z += mg.x_of_node + mg.y_of_node
+    fa = FlowAccumulator(mg)
+    fa.run_one_step()
+
+    with pytest.raises(ValueError):
+        ChiFinder(mg, min_drainage_area=1., reference_area=-1)
+
+
+def test_functions_with_Hex():
+    mg = HexModelGrid(10, 10)
+    z = mg.add_zeros("node", "topographic__elevation")
+    z += mg.x_of_node + mg.y_of_node
+    fa = FlowAccumulator(mg)
+    fa.run_one_step()
+
+    ch = ChiFinder(mg, min_drainage_area=1., reference_concavity=1.)
+    ch.calculate_chi()
```

Comment:
@SiccarPoint Can you add a more extensive test here? Right now this just verifies that it can function, not that it functions correctly...

Reply:
I'm closing this up, since these more comprehensive tests already exist as doctests within the component proper.

Reply:
(...but not for hex. I don't have time to thrash out a hex-specific solution!)

Reply:
sounds good. I'm not going to do it now either... But will merge once things pass.